src/modules/service-catalog/components/service-catalog-list-item/ServiceCatalogListItem.tsx

```diff
@@ -0,0 +1,88 @@
+import type { ServiceCatalogItem } from "../../data-types/ServiceCatalogItem";
+import ShapesIcon from "@zendeskgarden/svg-icons/src/16/shapes-fill.svg";
+import styled from "styled-components";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+
+const StyledItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+  gap: ${(props) => props.theme.space.sm};
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  border-radius: ${(props) => props.theme.borderRadii.md};
+  cursor: pointer;
+  padding: ${(props) => props.theme.space.md};
+  word-break: break-word;
+  border: ${(props) => props.theme.borders.sm}
+    ${(props) => getColorV8("grey", 300, props.theme)};
+  &:hover {
+    border: ${(props) => props.theme.borders.sm};
+    border-color: ${(props) => getColorV8("blue", 600, props.theme)} !important;
```

Comment:
Why do we need `!important`?

Reply:
I forgot to remove it after customization of Tiles component.